tools/lidar2image/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.21)
+project(lidar2image)
+
+find_package(Eigen3 REQUIRED)
+find_package(PCL 1.10 REQUIRED)
```

Thanks for your contribution. Is it necessary to use PCL 1.10? I tried Ubuntu 18.04 with default 1.8.1 also works.


```suggestion
find_package(PCL REQUIRED)
```